cmd/bb_worker/main.go

```diff
@@ -170,6 +173,18 @@ func main() {
 				log.Fatal("Failed to parse maximum execution timeout")
 			}
 
+			var devices []re_filesystem.Device
+			for _, device := range runnerConfiguration.Devices {
+				var stat unix.Stat_t
+				if err := unix.Stat(filepath.Join("/dev", device), &stat); err != nil {
+					log.Fatalf("Unable to stat device /dev/%#v", device)
```

Comment:
I suspect that this will print `Unable to stat device /dev/"null"`. It also doesn't print `err`. What are your thoughts on doing something like this?
```go
devicePath := filepath.Join("/dev", device)
if err := unix.Stat(devicePath), &stat); err != nil {
        log.Fatalf("Unable to stat device %#v: %s", devicePath, err)
}
```